src/utils.js

```diff
@@ -72,15 +72,15 @@ export const logSystemInfo = () => {
 
 /**
  * Gets the default instance of the `ApifyClient` class provided
- * <a href="https://apify.com/docs/sdk/apify-client-js/latest"
+ * <a href="https://sdk.apify.com/docs/api/apify#apifyclient"
```

Comment:
This should point to `apify-client` docs, not back to self.

Reply:
@apienx pls can you fix those, so that we can merge it?

Reply:
Oh, these are already fixed